Transportation/Bike/BikeHireDockingStation/doc/spec.md

```diff
@@ -124,24 +119,24 @@ A JSON Schema corresponding to this data model can be found [here](http://fiware
 
 Bike hire docking station real time data in Malaga
 
-	{
-	   "id": "malaga-bici-7"
-	   "type": "BikeHireDockingStation",
-	   "name": "07-Diputacion",
-	   "location": {
-		 "coordinates": [-4.43573, 36.699694],
-		  "type": "Point"
-	   },
-	   "availableBikeNumber": 18,
-	   "freeSlotNumber": 10,
-	   "address": {
-		  "streetAddress": "Paseo Antonio Banderas (Diputación)",
-		  "addressLocality": "Malaga",
-		  "addressCountry": "España"
-	   },
-	   "description": "Punto de alquiler de bicicletas próximo a Diputación",
-	   "dateModified": "2017-05-09T09:25:55.00Z"
-	}
+    {
```

Comment:
please do not make not substantive changes to existing data models.

Reply:
Sorry, this change was in for the docs to pass the checks.